### DISCUSSION TITLE
Everyone, support for multimodal models is coming soon. I am very excited!

### DISCUSSION THREAD

**Discussion by stevenlu137** [myshell-ai/AIlice#13] [Show and tell]
With the support of multimodal models, we can build richer feedback loops, such as iterative CAD modeling based on visual feedback. We will unlock a wave of future possibilities!

**Comment by Wallacerogersjr** [comment]
Look forward to anything new. This is awesome . Thank you

**Reply by stevenlu137** [reply]
I'm trying to create some modules that support multi-modality so that the model can actually see web pages, 3D models, local image files and screenshots. 
Because of the recent surprises brought by the miqu model, the progress of multi-modality has been a lot slower. The main focus will be on multimodal applications from now on.
